ml/README.md

```diff
@@ -0,0 +1,264 @@
+- `hosts to skip from training`: This parameter allows you to turn off anomaly detection for any child hosts on a parent host by defining those you would like to skip from training here. For example, a value like `dev-*` skips all hosts on a parent that begin with the "dev-" prefix. The default value of `!*` means "don't skip any".
+- `charts to skip from training`: This parameter allows you to exclude certain charts from anomaly detection by defining them here. By default, all charts, apart from a specific allow list of the typical basic Netdata charts, are excluded. If you have additional charts you would like to include for anomaly detection, you can add them here. **Note**: It is recommended to add charts in small groups and then measure any impact on performance before adding additional ones.
+
+## Charts
+
+Once enabled, the "Anomaly Detection" menu and charts will be available on the dashboard.
+
+![anomaly_detection_menu](https://user-images.githubusercontent.com/2178292/144255721-4568aabf-39c7-4855-bf1c-31b1d60e28e6.png)
+
+In terms of anomaly detection, the most interesting charts would be the `anomaly_detection.dimensions` and `anomaly_detection.anomaly_rate` ones, which hold the `anomalous` and `anomaly_rate` dimensions that show the overall number of dimensions considered anomalous at any time and the corresponding anomaly rate.
+
+- `anomaly_detection.dimensions`: Total count of dimensions considered anomalous or normal.
+- `anomaly_detection.dimensions`: Percentage of anomalous dimensions.
+- `anomaly_detection.detector_window`: The length of the active window used by the detector.
+- `anomaly_detection.detector_events`: Flags (0 or 1) to show when an anomaly event has been triggered by the detector.
+- `anomaly_detection.prediction_stats`: Diagnostic metrics relating to prediction time of anomaly detection.
+- `anomaly_detection.training_stats`: Diagnostic metrics relating to training time of anomaly detection.
+
+Below is an example of how these charts may look in the presence of an anomaly event.
+
+Initially we see a jump in `anomalous` dimensions:
+
+![anomalous](https://user-images.githubusercontent.com/2178292/144256036-c89fa768-5e5f-4278-9725-c67521c0d95e.png)
+
+And a corresponding jump in the `anomaly_rate`:
+
+![anomaly_rate](https://user-images.githubusercontent.com/2178292/144256071-7d157438-31f3-4b23-a795-0fd3b2e2e85c.png)
+
+After a short while the rolling node anomaly rate goes `above_threshold`, and once it stays above threshold for long enough a `new_anomaly_event` is created:
+
+![anomaly_event](https://user-images.githubusercontent.com/2178292/144256152-910b06ec-26b8-45b4-bcb7-4c2acdf9af15.png)
+
+## Glossary
```

Comment:
Does the sorting of the entries follow a certain logic? Intuitively, I would sort the glossary entries alphabetically.

Reply:
yep - good idea ill sort alpha

Reply:
oh actually, feature vector -> anomaly score -> anomaly bit -> anomaly rate -> anomaly detector -> anomaly event

actually does have a conceptual flow from smallest building block up to final node anomaly event.